sal_interfaces/Rotator/Rotator_Commands.xml

```diff
@@ -164,73 +97,6 @@
     </item>
 </SALCommand>
 
-<SALCommand>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_command_positionSet</EFDB_Topic>
-    <Alias>positionSet</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Description>Set the desired fixed position of the rotator. Issue the move command to actually move the rotator.</Description>
-    <item>
-        <EFDB_Name>angle</EFDB_Name>
-        <Description>angle</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>deg</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand>
-
-<SALCommand>
-    <Subsystem>Rotator</Subsystem>
-    <Version>3.5.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Rotator_command_moveConstantVelocity</EFDB_Topic>
```

Comment:
According to the engineering team, this command is useful for testing. So we should keep this.

Reply:
It is not supported by the CSC. Instead I offer a command-line script that allows you to command a ramp, which has the same effect.

Reply:
@tribeiro pointed me to this https://github.com/lsst-ts/ts_rotator/blob/00e19f5af74d83595bf104a908d6480b6b3095f2/python/lsst/ts/rotator/rotator_commander.py#L44-L47

Reply:
Exactly. Also the next command listed gives you a sinusoidal motion and we can add other shapes if they prove useful for testing.